lib/command/src/Obelisk/Command/Run.hs

```diff
@@ -108,15 +108,16 @@ runWatch = do
 
 -- | Relative paths to local packages of an obelisk project
 -- TODO a way to query this
-getLocalPkgs :: Applicative f => f [FilePath]
-getLocalPkgs = pure ["backend", "common", "frontend"]
+getLocalPkgs :: Applicative f => f [(FilePath, Maybe String)]
+getLocalPkgs = pure [("backend", Nothing), ("common", Nothing), ("frontend", Nothing)]
```

Comment:
This is technically the only committed use-case, yet the package-equals-folder assumption was
1) highly unexpected
2) unreported due to #597 
3) inconsistent with obelisk itself, since I ran into this whole thing when adding `.obelisk/impl/lib/route` to `getLocalPkgs` for some experiments, and it wouldn't find `obelisk-route.cabal`

Reply:
I've definitely run into other confusing issues with the Obelisk libraries having names different than the directories - independent of this I'd prefer they weren't like that.

Reply:
@alexfmpe Just curious: what's the use case for doing (3)? To have `ob run` reload when obelisk-route changes?

Reply:
@srid Yeah I was trying to get that in the same ghci(d) as the skeleton to avoid slow compilation cycles (several obelisk packages depend on `obelisk-route`. Didn't quite get to something that works correctly enough always because `obelisk-run` is one of those dependents and `ob run` uses it, so I ran into several weird things on my repl.